snippets/with_exclusions_external_data_and_config.js

```diff
@@ -0,0 +1,38 @@
+// if you don't specify a html file, the sniper will generate a div
+var app = require("ProtVista");
+var instance = new app({
+    el: yourDiv,
+    uniprotacc : 'P05067',
+    customDataSources: [
+        {
+            url: './data/externalFeatures_',
+            authority: 'myLab',
```

Comment:
Is this required for filtering? Am wondering whether "authority" is the correct term for it, maybe check with @SangyaP when she's back. Also is it possible to content to the tooltip as key/value pairs?